content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Add new driver support matrix
+description: "Document to add new delfin driver support matrix"
+weight: 20
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"]
+---
+
+Find below steps to add new delfin driver support matrix
+1. Clone the documentation repository (https://github.com/sodafoundation/documentation.git) 
```

Comment:
done

Reply:
Still looks like this in the [deploy preview](https://deploy-preview-140--soda-docs.netlify.app/support-matrix/delfin/add-driver-support/).
![image](https://user-images.githubusercontent.com/19162717/109102914-e0a58e00-774f-11eb-9812-7cc7c2f20ef9.png)

Reply:
I believe this is because the numbered list expects `1.` and not `1)`
For a numbered list the number should be at the start of the line, no spaces and should be the number followed by a dot. 
At the end of a line to go to a new line you can either place two spaces or use `<br />`

Reply:
I can see the same issue below as well @Shruthi-1MN 
![image](https://user-images.githubusercontent.com/19162717/109103253-8c4ede00-7750-11eb-8fa1-15ee1e6f19f9.png)